snmp/tests/test_e2e_core_vs_python.py

```diff
@@ -21,7 +21,6 @@
 
 ASSERT_VALUE_METRICS = [
     'snmp.devices_monitored',
-    'datadog.snmp.submitted_metrics',
```

Comment:
Note for reviewers: Removing `datadog.snmp.submitted_metrics` from ASSERT_VALUE_METRICS since it's hard to maintain. The submitted metrics count if different in many cases due to some metrics being only send from corecheck.

`ASSERT_VALUE_METRICS` is already overrode in many cases anyway: https://github.com/DataDog/integrations-core/pull/11781/files